config/hocon-mp/src/main/resources/META-INF/native-image/io.helidon.config/helidon-config-hocon-mp/resource-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "resources": [
+        {
+            "pattern": "mp-meta-config.yaml"
```

Comment:
This should be part of module containing the meta configuration support. If we want to support meta configuration in HOCON/json, then this should contain those files. YAML should be part of yaml support.

Reply:
Should I just remove this, not really sure what this file is for?

Reply:
Removed